src/helpers/prepare-queued-pr-for-merge.ts

```diff
@@ -46,9 +46,9 @@ export const updatePrWithMainline = async (pullRequest: PullRequest | SimplePull
   } catch (error) {
     if ((error as GithubError).status === 204) {
       core.info('The first PR in the queue is already up to date!');
-    }
-    if ((error as GithubError).status === 409) {
-      core.info('The first PR in the queue has a merge conflict.');
+    } else if ((error as GithubError).status >= 400) {
+      if ((error as GithubError).status === 409) core.error('The first PR in the queue has a merge conflict.');
+      else core.error((error as GithubError).message);
```

Comment:
So we already have a generic error handler [here](https://github.com/ExpediaGroup/github-helpers/blob/main/src/main.ts#L27). Will this do anything different?

Reply:
nah, should be interpreted in the same way internally. just saw we used `core.error` in some places in the code

Reply:
I believe we only use `core.error` when there is a specific error to log, not a generic one. Could be wrong tho!